RELEASE.md

```diff
@@ -0,0 +1,243 @@
+# A guide to making a release
+
+This guide collects the steps we do in SetFit to make a release on PyPI. They result from hard-learned lessons (mostly on Transformers) and while following this guide is completely optional, it’s strongly recommended to do so. 🙂
+
+### Preparation
+
+To be able to make a release for a given project, you’ll need an account on [PyPI](https://pypi.org/) and on [Test PyPI](https://test.pypi.org/). If you are making a release for an existing project, your username will need to be added to that project by one of the current maintainers on PypI. Note that we strongly recommend enabling two-factor authentication on PyPI.
+
+You will also need to install twine in your Python environment with `pip install twine`.
+
+## Minor or major releases
+
+Follow this guide when you are release a new version of your package (not a patch). For patches go to the section on patch releases below.
+
+### Step 1: Cut a release branch
+
+This will ensure no one merges anything new in the upcoming release while you’re working on this. We recommend naming it `v{major}.{minor}-release` so you can reuse that same branch for patch releases.
+
+You can create the branch and push it with
+
+```bash
+git checkout -b v{major}.{minor}-release
+git push -u origin v{major}.{minor}-release
+```
+
+**Make sure you are at the right commit on the main branch before executing this and do not forget to pull the main branch if you want to branch from the last commit!**
+
+### Step 2: Prepare the release notes
+
+We have [a space](https://huggingface.co/spaces/lysandre/github-release) to help you with that. You will need a [GitHub token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) with the following scopes:
+
+![](assets/scopes.png)
```

Comment:
I tend to drag and drop or paste files into GitHub to do this. This results in:

`![image](https://user-images.githubusercontent.com/37621491/232101171-bc99b18f-fb75-4f4c-b16c-aaccda4ae2b1.png)`

which produces

![image](https://user-images.githubusercontent.com/37621491/232101171-bc99b18f-fb75-4f4c-b16c-aaccda4ae2b1.png)

Then I don't have to use an assets folder. Is there any reason for using the assets folder? Just so you know it'll always be there?

Reply:
Oh this is a cool trick! Yeah, I usually have an `assets` folder for images (especially if they will change), but in this case a screenshot hosted on GitHub is much better

Reply:
Done in https://github.com/huggingface/setfit/pull/357/commits/6a67da0723f5578203a7ee61097eeff06c4d290b